scripts/build/build_macos.sh

```diff
@@ -8,22 +8,22 @@
 set -euo pipefail
 set -x
 
-dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Please ignore auto-formatting changes for now. My IDE is all messed up. I will fix it up at the very end.